RestaurantReservationSystem/Seeders/TableSeeder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Seeders;
+
+public class TableSeeder
```

Comment:
seems like all seeder classes have the same function signature, lets have them all implement an `IDataSeeder` interface and use generics.